### PR TITLE
feat: add Elfsight referral link + instructions

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-12082](https://app.clickup.com/t/2645698/DEV-12082)

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-12082 Add elfsight referal link in widget description](https://app.clickup.com/t/2645698/DEV-12082)

**Review by doubleppereira — APPROVED** [review]

File: translations/en_US.json [hunk 1]
```diff
@@ -180,7 +180,7 @@
   "shuffle_title": "Profiles",
   "token_asset_type_video": "VIDEO",
   "beta_warning": "You are on <strong>LUKSO Testnet</strong>. Do not send real LYX to any profiles made on this network!<br /><strong>If you are looking for LUKSO Mainnet, please use the switch in the footer.</strong>",
-  "edit_widget_elfsight_description": "Use this widget to display Elfsight on your grid.",
+  "edit_widget_elfsight_description": "Here you can add custom widgets like a instagram feed, or calendar from <a class=\"text-purple-51 underline hover:text-purple-41\" href=\"https://elfsight.com/widgets/?ref=e9caec1c-1acd-423a-9a19-d18bd676fd00\" target=\"_blank\">Elfsight.com</a>.",
```
Comment (JeneaVranceanu) [inline translations/en_US.json:183]:
```suggestion
  "edit_widget_elfsight_description": "Here you can add custom widgets like an instagram feed, or calendar from <a class=\"text-purple-51 underline hover:text-purple-41\" href=\"https://elfsight.com/widgets/?ref=e9caec1c-1acd-423a-9a19-d18bd676fd00\" target=\"_blank\">Elfsight.com</a>.",
```

File: translations/en_US.json [hunk 2]
```diff
@@ -298,7 +298,7 @@
   "token_details_buy_lyx": "Buy LYX",
   "footer_terms_text": "Terms & Conditions",
   "followed_by_text": "Followed by {names} {othersCount, plural,\n      =0 {you follow}\n      =1 {and 1 other you follow}\n      other {and # others you follow}\n    }",
-  "add_widget_elfsight_description": "Use this widget to display Elfsight on your grid.",
+  "add_widget_elfsight_description": "Here you can add custom widgets like a instagram feed, or calendar from <a class=\"text-purple-51 underline hover:text-purple-41\" href=\"https://elfsight.com/widgets/?ref=e9caec1c-1acd-423a-9a19-d18bd676fd00\" target=\"_blank\">Elfsight.com</a>.",
```
Comment (JeneaVranceanu) [inline translations/en_US.json:301]:
```suggestion
  "add_widget_elfsight_description": "Here you can add custom widgets like an instagram feed, or calendar from <a class=\"text-purple-51 underline hover:text-purple-41\" href=\"https://elfsight.com/widgets/?ref=e9caec1c-1acd-423a-9a19-d18bd676fd00\" target=\"_blank\">Elfsight.com</a>.",
```

**Review by JeneaVranceanu — APPROVED** [review]
A few typos.

**Review by JeneaVranceanu — APPROVED** [review]

**Review by richtera — APPROVED** [review]

**Pull request merged**